Gruntfile.js

```diff
@@ -182,7 +182,7 @@ module.exports = function (grunt) {
                     "<%= paths.node %>/mocha/mocha.js",
                     "<%= paths.node %>/mocha/mocha.css",
                     "<%= paths.node %>/chai/chai.js",
-                    "<%= paths.node %>/mock-socket/dist/mock-socket.js",
+                    "<%= paths.bower %>/mock-socket/dist/mock-socket.js",
```

Comment:
Why? Is it a mistake or there's a reason to add it to Bower again?
*Edit*: Ok, just saw your message on Gitter. Still, can you leave it in node dependencies instead of Bower's?

Reply:
I'm pretty sure it would conflict with the newer version we have. I don't want to start fixing the old tests since they'll be deleted as soon as the Vue side is ready. Then we can fully drop bower too.

Reply:
Ah, you kept the newer version. Ok, then.